sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/HandleItem.java

```diff
@@ -80,7 +80,7 @@ private AccessRightListWrapper(@JacksonXmlProperty(localName = "AccessRight") Li
      * The AccessRightList property.
      */
     @JsonProperty(value = "AccessRightList")
-    private AccessRightListWrapper accessRightList;
+    private AccessRightListWrapper accessRights;
```

Comment:
Was there a central Swagger update to rename this property? I don't see anything in Swagger transforms to do this and want to make sure this wasn't a manual change that won't pass CI.

Reply:
This was a manual change. There's two HandleItem.java, one that's in the models folder and one that's in the implementation folder. The one in the models folder is the handwritten one.